src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -461,8 +463,8 @@ class SaagieClient {
                     return newJobVersion
                 }
             }
-        } else if (configuration?.job?.id) {
-            Request lisJobVersion  = saagieUtils.getListVersionForJobRequest(configuration?.job?.id)
+        } else if (job?.id) {
+            Request lisJobVersion  = saagieUtils.getListVersionForJobRequest(job?.id)
```

Comment:
there's a fail in the variable name, I think you wanted to name it `lisJobVersion`